### DISCUSSION TITLE
Not able to get Filebeat logs into Elasticsearch index

### DISCUSSION THREAD

**Discussion by vravru135** [learndevopsonline/learndevopsonline#1877] [Q&A]
Mar 29 18:44:25 ip-172-31-42-96 filebeat[11578]: {"log.level":"error","@timestamp":"2024-03-29T18:44:25.720Z","log.logger":"publisher_pipeline_output","log.origin":{"function":"[github.com/elastic/beats/v7/libbeat/publisher/pipeline.(*netClientWorker).run](http://github.com/elastic/beats/v7/libbeat/publisher/pipeline.(*netClientWorker).run)","[file.name](http://file.name/)":"pipeline/client_worker.go","file.line":148},**"message":`"Failed to connect to backoff(elasticsearch(http://172.31.27.209:5044)): Get \"http://172.31.27.209:5044/\": read tcp 172.31.42.96:56640->[172.31.27.209:5044](http://172.31.27.209:5044/): read: connection reset by peer","[service.name](http://service.name/)":"filebeat","ecs.version":"1.6.0"}**
`

`"Failed to connect to backoff(elasticsearch(http://172.31.27.209:5044)): Get \"http://172.31.27.209:5044/\": read tcp 172.31.42.96:56640->[172.31.27.209:5044](http://172.31.27.209:5044/): read: connection reset by peer","[service.name](http://service.name/)":"filebeat","ecs.version":"1.6.0"}
`

**Comment by chaithanya1812** [comment]
Hi @vravru135 , can you join now sir is available..
